samples/boards/mec15xxevb_assy6853/power_management/src/power_mgmt.c

```diff
@@ -256,21 +255,24 @@ static void resume_all_tasks(void)
 	k_thread_resume(&thread_b_id);
 }
 
-static struct pm_notifier notifier = {
-	.state_entry = notify_pm_state_entry,
-	.state_exit = notify_pm_state_exit,
-};
+PM_NOTIFIER_DEFINE(test_light_sleep, (PM_STATE_ENTRY|PM_STATE_EXIT), notify_pm_state, NULL);
+PM_NOTIFIER_DEFINE(test_deep_sleep, (PM_STATE_ENTRY|PM_STATE_EXIT), notify_pm_state, NULL);
```

Comment:
these variables names could have something that say that they are notifiers. We have
`test_light_sleep` and `light_sleep` variables in this file